x/emissions/keeper/msgserver/msg_server_test.go

```diff
@@ -257,3 +257,44 @@ func (s *KeeperTestSuite) TestCreateSeveralTopics() {
 	s.Require().NotNil(result)
 	s.Require().Equal(initialTopicId+2, result)
 }
+
+func (s *KeeperTestSuite) inEpsilon(value alloraMath.Dec, target string, epsilon string) {
```

Comment:
can we make a test util folder and move these inEpsilon functions into a separate package if we need them in multiple places in the tests, seems like bad practice to have duplicate code

Reply:
updated! Check it out. There is a little weirdness around not being able to call s.Require() when initializing the alloratestutil package INSIDE initializing an outer package, so I switched to this function contructor pattern

Reply:
https://github.com/allora-network/allora-chain/pull/324

Reply:
merge